zivid_description/urdf/macros/zivid_camera.xacro

```diff
@@ -1,25 +1,25 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
-    <xacro:macro name="zivid_camera" params="prefix">
-        <!-- Zivid Properties -->
-        <xacro:property name="M_PI" value="3.1415926535897931" />
+    <!-- Properties -->
+    <material name="zivid_gray">
+        <color rgba="0.25 0.25 0.25 1"/>
+    </material>
 
+    <xacro:macro name="zivid_camera" params="prefix">
         <!-- Zivid Base Link -->
         <link name="${prefix}base_link">
             <!-- Visuals -->
             <visual>
-                <origin xyz="-0.0030 -0.0758 0.0445" rpy="${0.5*M_PI} 0 ${0.5*M_PI}"/>
-                <material name="blue">
-                    <color rgba="0.25 0.25 0.25 1"/>
-                </material>
+                <origin xyz="-0.0030 -0.0758 0.0445" rpy="${0.5*pi} 0 ${0.5*pi}"/>
```

Comment:
Nice, I much prefer using the `pi` python math constant directly over `M_PI` in the math expressions 👍